pkg/registry/registry_antrea.csv

```diff
@@ -54,3 +54,5 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 152,flowEndSecondsFromDestinationNode,unsigned32,,current,,,,,,,,56506,
 153,egressName,string,,current,,,,,,,,56506,
 154,egressIP,string,,current,,,,,,,,56506,
+155,isL7,string,,current,,,,,,,,56506,
+156,httpVals,string,,current,,,,,,,,56506,
```

Comment:
a similar question: is this going to be some JSON object? What will be in this object?

Reply:
It is not going to be a json value, it will be a string containing all l7 http fields separated by a delimiter.